unit2.js

```diff
@@ -0,0 +1,94 @@
+// function getOption() {
```

Comment:
There are few things to improve/review from this assessment. Here are feedbacks: 
1. Styling could be done better and follow guideline provided in the rubric. Missing h1 tag in the header, wrong dimensions for elements, and lack of flex or grid are among them. Please review and practice CSS and reach out to me if you need any help. 
2. Functionality could improve with feedbacks provided below.  
3. When a movie is selected, it shows its title but not the description. 
4. Comment function ALMOST would have worked by adding few things in it. 

Please review and practice this assessment as it is good benchmark for things that we will learn and create in the class. Please reach out to instructional team if you need any help or have questions.